jekyll/__glossary/layer_caching.md

```diff
@@ -0,0 +1,5 @@
+---
+term: Docker Layer Caching
+---
+
+The CircleCI Docker Layer Caching feature allows builds to reuse Docker image layers from previous builds. Images and layers are stored in separate volumes in the cloud and are not shared between projects. Layers may only be used by builds from the same project. See the [Docker Layer Caching]({{ site.baseurl }}/2.0/docker-layer-caching/) and [Running Docker Commands](({{ site.baseurl }}/2.0/building-docker-images/) documents for additional information about how the secure environment is created. 
```

Comment:
1st sentence: Trim the fat

"Docker layer caching allows builds to reuse Docker image layers from previous builds."

2nd sentence: The first sentence mentioned "image layers" and this sentence is talking about "images and layers". Are these the same or distinct?

3rd sentence: Purpose first.

"For more information about environment security, see [Docker Layer Caching] and [Running Docker Commands]."

Reply:
I will leave the qualification in the first sentence because this is a feature of CircleCI that is unique to us. 
Good catch, second and third sentences fixed.